docs/csharp/language-reference/keywords/typeof.md

```diff
@@ -25,11 +25,22 @@ System.Type type = i.GetType();
   
  The `typeof` operator cannot be overloaded.  
   
- The `typeof` operator can also be used on open generic types. Types with more than one type parameter must have the appropriate number of commas in the specification. The following example shows how to determine whether the return type of a method is a generic <xref:System.Collections.Generic.IEnumerable%601>. Assume that method is an instance of a MethodInfo type:  
+ The `typeof` operator can also be used on open generic types. Types with more than one type parameter must have the appropriate number of commas in the specification. The following example shows how to determine whether the return type of a method is a generic <xref:System.Collections.Generic.IEnumerable%601>. `GetInterface` will return null if the return type is not a IEnumerable<> generic type.
```

Comment:
\`GetInterface\` --> \<xref:System.Type.GetInterface%2A?displayProperty=nameWithType> 
null --> \`null\`
is not a --> is not an
The second IEnumerable\<> should either be fenced or an xref (like the first one)